tests/integration/helper.py

```diff
@@ -4,36 +4,46 @@
 
 from ..helper import cd
 
-__all__ = ["add_post_without_text", "append_config", "cd", "patch_config"]
+__all__ = ["add_post_without_text", "create_simple_post", "append_config", "cd", "patch_config"]
```

Comment:
I think for overall readability it is the easiest to have these ordered alphabetically.